diff --git a/cmod/phy_top/include/phy_spec.h b/cmod/phy_top/include/phy_spec.h
index a4dcf10c..adfb4f34 100644
--- a/cmod/phy_top/include/phy_spec.h
+++ b/cmod/phy_top/include/phy_spec.h
@@ -30,10 +30,10 @@ namespace phy_spec {
 
     // Timeout cycles
     const int kResetCycles = 10000;
-    const int kStateTimeoutCycles = 10000;
+    const int kStateTimeoutCycles = 6400000;
     const int kLinkErrorMinCycles = 10000;
     const int kSbTrainingPhaseCycles = 384;
-    const int kSbTrainingTimeoutCycles = 8 * kSbTrainingPhaseCycles;
+    const int kSbTrainingTimeoutCycles = 6400000;
 
     // Mainband
     const int kNumLanes = 16;
